java/dataset/src/main/java/org/apache/arrow/dataset/substrait/AceroSubstraitConsumer.java

```diff
@@ -90,6 +91,15 @@ public ArrowReader runQuery(ByteBuffer plan, Map<String, ArrowReader> namedTable
     return execute(plan, namedTables);
   }
 
+  public List<String> runDeserializeExpressions(ByteBuffer plan) {
```

Comment:
Docstrings? I don't think we want these at all though?

Reply:
Deleted